Bottom: 549c3154a3ff1bd74e777e38cbc3ad9f913225bc
Top:    b89ef72d19fbdc513f2b40e916dba08a8bc02fb3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 19:44:23 +0100

Refresh of temp

---

diff --git a/src/draw.c b/src/draw.c
index 560e094..4340526 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1693,9 +1693,12 @@ EraseElementPinsAndPads (ElementTypePtr Element)
 void
 EraseElementName (ElementTypePtr Element)
 {
+  assert (Gathering);
+
   if (TEST_FLAG (HIDENAMEFLAG, Element))
     return;
-  DrawTextLowLevel (&ELEMENT_TEXT (PCB, Element), PCB->minSlk);
+
+  AddPart (&ELEMENT_TEXT (PCB, Element));
 }
